packages/access-client/src/agent.js

```diff
@@ -642,23 +907,28 @@ export class Agent {
   async invoke(cap, options) {
     const space = options.with || this.currentSpace()
```

Comment:
long term it feels weird that a generic `invoke` here would try to be aware of currentSpace (because it could be invoking something unrelated to spaces)

Reply:
Sending extra proofs related to spaces could leak information about the end-user they don't expect